src/_guides/language/language-tour.md

```diff
@@ -2766,6 +2766,12 @@ which returns a [`Type`][] object.
 print('The type of a is ${a.runtimeType}');
 ```
 
+{{site.alert.warn}}
+  Use a [type test operator][] rather than `runtimeType` to test an object's type.
+  In production environments, the test `object is string` is more stable
+  than the test `object.runtimeType == string`.
```

Comment:
What do you think about using `Type` here instead of `string`?

For example:

```suggestion
  Use a [type test operator][] rather than `runtimeType`
  to test an object's type.
  In production environments, the test `object is Type` is more stable
  than the test `object.runtimeType == Type`.
```